src/client/job.rs

```diff
@@ -34,212 +24,7 @@ pub async fn get_worker_map(connection: &mut ClientConnection) -> anyhow::Result
-
-    let mut n_tasks = job.info.n_tasks.to_string();
-    if let JobType::Array(array_def) = &job.job_type {
-        n_tasks.push_str(&format!("; Ids: {}", array_def));
-    }
-
-    rows.push(vec!["Tasks".cell().bold(true), n_tasks.cell()]);
-    rows.push(vec![
-        "Workers".cell().bold(true),
-        format_job_workers(&job, &worker_map).cell(),
-    ]);
-
-    let resources = cpu_request_to_string(job.resources.cpus());
-
-    rows.push(vec![
-        "Resources".cell().bold(true),
-        if job.pin {
-            format!("{} [pin]", resources)
-        } else {
-            resources
-        }
-        .cell(),
-    ]);
-
-    rows.push(vec!["Priority".cell().bold(true), job.priority.cell()]);
-
-    let program_def = job.program_def;
-    rows.push(vec![
-        "Command".cell().bold(true),
-        program_def
-            .args
-            .iter()
-            .map(|x| x.to_string())
-            .collect::<Vec<String>>()
-            .join("\n")
-            .cell(),
-    ]);
-    rows.push(vec![
-        "Stdout".cell().bold(true),
-        stdio_to_cell(&program_def.stdout),
-    ]);
-    rows.push(vec![
-        "Stderr".cell().bold(true),
-        stdio_to_cell(&program_def.stderr),
-    ]);
-    let mut env_vars: Vec<(_, _)> = program_def
-        .env
-        .iter()
-        .filter(|(k, _)| !is_hq_env(k))
-        .collect();
-    env_vars.sort_by_key(|item| item.0);
-    rows.push(vec![
-        "Environment".cell().bold(true),
-        env_vars
-            .into_iter()
-            .map(|(k, v)| format!("{}={}", k, v))
-            .collect::<Vec<_>>()
-            .join("\n")
-            .cell(),
-    ]);
-
-    rows.push(vec![
-        "Working Dir".cell().bold(true),
-        program_def
-            .cwd
-            .map(|cwd| cwd.display().to_string())
-            .unwrap()
-            .cell(),
-    ]);
-
-    rows.push(vec![
-        "Submission date".cell().bold(true),
-        job.submission_date.round_subsecs(0).cell(),
-    ]);
-
-    rows.push(vec![
-        "Task time limit".cell().bold(true),
-        job.time_limit
-            .map(|duration| humantime::format_duration(duration).to_string())
-            .unwrap_or_else(|| "None".to_string())
-            .cell(),
-    ]);
-
-    rows.push(vec![
-        "Makespan".cell().bold(true),
-        human_duration(job.completion_date_or_now - job.submission_date).cell(),
-    ]);
-
-    let table = rows.table().color_choice(gsettings.color_policy());
-    assert!(print_stdout(table).is_ok());
-
-    if !job.tasks.is_empty() {
-        print_job_tasks(
-            gsettings,
-            job.completion_date_or_now,
-            job.tasks,
-            show_tasks,
-            &job.info.counters,
-            &worker_map,
-        );
-    }
-}
-
-const MAX_DISPLAYED_WORKERS: usize = 2;
-
-fn format_job_workers(job: &JobDetail, worker_map: &WorkerMap) -> String {
+pub fn format_job_workers(job: &JobDetail, worker_map: &WorkerMap) -> String {
```

Comment:
Let's also move these `format_*` functions to `cli.rs`. If you're using IntelliJ, you can move them using `F6`.